x-pack/plugins/actions/server/types.ts

```diff
@@ -81,6 +81,7 @@ export interface ActionResult<Config extends ActionTypeConfig = ActionTypeConfig
   config?: Config;
   isPreconfigured: boolean;
   isDeprecated: boolean;
+  isSystemAction: boolean;
```

Comment:
It would be great to start documenting at least public interfaces that are exported from the Framework -- for us on the solution side it's often hard to navigate the Framework's code, but we need to do it from time to time.

Here we could add a JSDoc comment with a description of the fields and a link to https://github.com/elastic/kibana/issues/160367.

Reply:
Good point! I will do it in the following PRs as I will probably introduce more types.